docs/user/azure/install.md

```diff
@@ -0,0 +1,64 @@
+# Cluster Installation
+
+At this point, you are ready to perform the OpenShift installation. See below for an example of an
+IPI install.
+
+## Example: Installer-Provided Infrastructure (IPI)
+
+The steps for performing an IPI-based install are outlined [here][cloud-install]. Following this guide you may begin at
+Step 3: Download the Installer.
+
+### Create Configuration
+
+```console
+[~]$ openshift-install create install-config
+? SSH Public Key /home/user_id/.ssh/id_rsa.pub
+? Platform azure
+? azure subscription id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+? azure tenant id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+? azure service principal client id xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+? azure service principal client secret xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+? Region centralus
+? Base Domain example.com
+? Cluster Name test
+? Pull Secret [? for help]
+```
+
+### Create Cluster
+
+```console
+[~]$ openshift-install create cluster
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
+INFO API v1.11.0+85a0623 up
+INFO Waiting up to 30m0s for the bootstrap-complete event...
+INFO Destroying the bootstrap resources...
+INTO Waiting up to 30m0s for the cluster at https://api.test.example.com:6443 to initialize...
+INFO Waiting up to 10m0s for the openshift-console route to be created...
+INFO Install complete!
+INFO To access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/home/user/auth/kubeconfig'
+INFO Access the OpenShift web-console here: https://console-openshift-console.apps.test.example.com
+INFO Login to the console with user: kubeadmin, password: 5char-5char-5char-5char
+```
+
+### Running Cluster
+
+In your subscription, there will be a new, Resource Group for your cluster:
+
+![Cluster Resource Group](images/install_resource_group.png)
+
+In The resource group, there will be 6 running Virtual Machines running:
+
+![Virtual Machines instances after install](images/install_nodes.png)
+
+**TODO** update information on the encrypted disks as soon as we have those.
```

Comment:
I'd rather track this in an issue or other RFE, vs. having a TODO in the docs.  There's nothing fundamentally broken about the lack of disk encryption.  It would obviously be nice to have, as would a number of other improvements, and an RFE tracker somewhere would give us a place to talk through plans, post status updates, etc.

Reply:
> There's nothing fundamentally broken about the lack of disk encryption. It would obviously be nice to have, as would a number of other improvements,

disk encryption for control-plane is not an RFE but a requirement. There's already a tracker in JIRA https://jira.coreos.com/browse/CORS-1098

But this is to track most a placeholder to where the information needs to go.

But I can remove these TODO for now.